test/integration/policy-limitmemory_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+const policyLimitMemoryURL = "https://raw.githubusercontent.com/stolostron/policy-collection/main/stable/SC-System-and-Communications-Protection/policy-limitmemory.yaml"
+const policyLimitMemoryName = "policy-limitmemory"
+const limitRangeName = "mem-limit-range"
+
+var limitMemoryLabel = Label("policy-collection", "stable", "BVT")
```

Comment:
Looks good. The only thing is I would suggest not to put BVT label on it. The config policy controller has been already tested by other test cases. It's safe to leave this test case for SVT.